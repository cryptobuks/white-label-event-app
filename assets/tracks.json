[
  {
    "id": "innovation",
    "title": "Innovation"
  },
  {
    "id": "pbd",
    "title": "Product & Business development"
  },
  {
    "id": "newTech",
    "title": "New Tech"
  },
  {
    "id": "ai",
    "title": "AI"
  },
  {
    "id": "agile",
    "title": "Agile"
  }
]