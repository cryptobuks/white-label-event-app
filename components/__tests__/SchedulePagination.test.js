import React from 'react';
import renderer from 'react-test-renderer';
import SchedulePagination from '../SchedulePagination';
import tracks from '../../assets/tracks.json';

describe('SchedulePagination', () => {
  it('renders correctly', () => {
    const comp = (
      <SchedulePagination
        index={0}
        total={5}
        tracks={tracks}
        onNextTap={() => {}}
        onLongPress={() => {}}
      />
    );

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
