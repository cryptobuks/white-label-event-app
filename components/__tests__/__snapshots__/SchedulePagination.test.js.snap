// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SchedulePagination renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "rgba(0,0,0,.5)",
      "flex": 1,
      "flexDirection": "row",
      "height": 104,
      "justifyContent": "center",
      "paddingTop": 64,
      "position": "absolute",
      "width": 672,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 14,
            "fontWeight": "bold",
            "width": 224,
          },
          Object {
            "opacity": 0.4,
            "textAlign": "right",
          },
        ]
      }
    >
      
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={undefined}
    testID={undefined}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "column",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 14,
              "fontWeight": "bold",
              "width": 224,
            },
            Object {
              "marginHorizontal": 24,
              "textAlign": "center",
            },
          ]
        }
      >
        Innovation
      </Text>
      <View
        style={
          Object {
            "backgroundColor": "#e54cfe",
            "height": 2,
            "marginTop": 8,
            "width": 20,
          }
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 14,
            "fontWeight": "bold",
            "width": 224,
          },
          Object {
            "opacity": 0.4,
            "textAlign": "left",
          },
        ]
      }
    >
      Product & Business development
    </Text>
  </View>
  <View
    style={
      Array [
        undefined,
        Object {
          "backgroundColor": "transparent",
          "left": 154,
          "position": "absolute",
          "top": 56,
        },
      ]
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 48,
          "width": 144,
        }
      }
      testID={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "transparent",
              "flex": 1,
            },
            Object {
              "height": 48,
              "width": 144,
            },
          ]
        }
      />
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "left": 214,
          },
          Object {
            "height": 48,
            "width": 144,
          },
        ]
      }
      testID={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "transparent",
              "flex": 1,
            },
            Object {
              "height": 48,
              "width": 144,
            },
          ]
        }
      />
    </View>
  </View>
</View>
`;
